tycho-surefire/tycho-surefire-plugin/src/main/java/org/eclipse/tycho/surefire/AbstractEclipseTestMojo.java

```diff
@@ -528,6 +528,18 @@ public abstract class AbstractEclipseTestMojo extends AbstractTestMojo {
     @Parameter
     private int reactorConcurrencyLevel;
 
+    public enum ClassLoaderOrder {
+        booterFirst, testProbeFirst
+    }
+
+    /**
+     * The test runtime is configured with a composite class loader. This defines the order in which
+     * the loaders are searched. This may need to be configured depending on the classpath of the
+     * project. Available values "booterFirst" and "testProbeFirst"
+     */
+    @Parameter(defaultValue = "booterFirst")
```

Comment:
Always good to be tell what's the default value in the docs.

```suggestion
    /**
     * The test runtime is configured with a composite class loader.
     * This defines the order in which the loaders are searched, and it may need to be configured depending on the classpath of the project.
     * <p>
     * Available values are <code>booterFirst</code>, <code>testProbeFirst</code>.
     * Defaults to <code>booterFirst</code>.
     */
    @Parameter(defaultValue = "booterFirst")
```